openhtf/io/output/mfg_inspector.py

```diff
@@ -239,7 +239,7 @@ def _ExtractParameters(record, testrun, used_parameter_names):
         # Just a plain ol' value.
         if isinstance(value, numbers.Number):
           testrun_param.numeric_value = float(value)
-        else:
+        elif value is not None:
```

Comment:
Actually, I'd check `measurement.measured_value.is_value_set` instead, since if the parameter is actually set to None we want that to be an error. We only want purely unset ones to show up with no value set.

Also, the bug this PR references is that the unset measurements show up as a failure code, which means the real bug is that the code up above checking if the measurement has a `measured_value` isn't working. It should only get to here if it was actually set to None.

I'd say that `measurement.measured_value.is_value_set` should be checked rather than `measurement.measured_value` since the `measured_value` will always be there and neither `MeasuredValue` nor `DimensionedMeasuredValue` have `__nonzero__` implemented.